src/html/dom-module-name-or-is.ts

```diff
@@ -49,18 +49,32 @@ class DomModuleNameOrIs extends HtmlRule {
         p.hasTagName('dom-module'), p.OR(p.hasAttr('is'), p.hasAttr('name')));
     const badModules = dom5.queryAll(document.ast, badModule);
     for (const domModule of badModules) {
+      const isFixable =
+          !(dom5.getAttribute(domModule, 'is') !== null &&
+            dom5.getAttribute(domModule, 'name') !== null);
       for (const badAttr of ['is', 'name']) {
         const attr = dom5.getAttribute(domModule, badAttr);
-        if (attr != null) {
+        if (attr !== null) {
```

Comment:
Not your bug, but prefer to check and continue rather than adding another layer of indentation. So:

```typescript
if (attr === null) {
  continue;
}
```